### ISSUE TITLE
frame WebViewes eat keyboard input

### ISSUE THREAD

**Issue by alex19EP** [microsoft/vscode#104329]
Issue Type: <b>Bug</b>

1. open Release Notes
2. press alt to open the file menu

## what's happening.

menu dosn't opens.

## what should happen.

menu opens, focus moves

VS Code version: Code - Insiders 1.48.0-insider (2277c8e2a3e1cc630a6397301ba54a1dccd8a60d, 2020-08-07T11:32:09.993Z)
OS version: Windows_NT x64 10.0.19042

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-7700K CPU @ 4.20GHz (8 x 4200)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>protected_video_decode: enabled<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>viz_display_compositor: enabled_on<br>viz_hit_test_surface_layer: disabled_off_ok<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|15.86GB (7.90GB free)|
|Process Argv||
|Screen Reader|yes|
|VM|25%|
</details><details><summary>Extensions (22)</summary>

Extension|Author (truncated)|Version
---|---|---
github-markdown-preview|bie|0.0.2
markdown-checkbox|bie|0.1.3
markdown-emoji|bie|0.0.9
markdown-preview-github-styles|bie|0.1.6
markdown-yaml-preamble|bie|0.0.4
vscode-astyle|chi|0.9.0
vscode-markdownlint|Dav|0.36.2
githistory|don|0.6.9
EditorConfig|Edi|0.15.1
vscode-pull-request-github|Git|0.18.1
vscode-language-pack-ru|MS-|1.48.2
python|ms-|2020.7.96456
remote-ssh|ms-|0.51.0
remote-ssh-edit|ms-|0.51.0
remote-wsl|ms-|0.44.4
cmake-tools|ms-|1.4.1
cpptools|ms-|0.29.0
vsliveshare|ms-|1.0.2478
vsliveshare-audio|ms-|0.1.85
vsliveshare-pack|ms-|0.4.0
code-spell-checker|str|1.9.0
code-spell-checker-russian|str|0.2.4

</details>

**Comment by isidorn** [comment]
I can reproduce this - just open any markdown preview and `alt+f` will not open the file menu. 
I think we should fix this for this release, as it is a breaking change (in last stable I can nicely press `alt+f` to open the file menu). Due to that I am assigning this to July and @mjbvz can move it out if he would prefer to fix it next milestone

Also fyi @sbatten for custom menus

**Comment by mjbvz** [comment]
@sbatten Has the way the menus handle mnemonics changed this iteration? The webview still seem to be dispatching the events but they aren't being handled by anyone

**Comment by sbatten** [comment]
@mjbvz nope no changes that i know of

**Comment by sbatten** [comment]
should be handled here https://github.com/microsoft/vscode/blob/4a1bcdafe97820778ba86f40c225dbb40585c8f0/src/vs/base/browser/ui/menu/menubar.ts#L1020

**Comment by mjbvz** [comment]
`alt+f` is working for me and @sbatten in insiders on windows. 

For just pressing `alt` to show the mnemonics, this does not seem like a regression from 1.47 in my testing but it was working in 1.46

**Comment by sbatten** [comment]
I was just testing this out, I see no new behavioral differences between now and 1.46

1. Alt does not focus the custom menubar with release notes focused. We do not get the alt event
2. Alt+<mnemonic> does open the resepective menu with release notes focused.

**Comment by alex19EP** [comment]
hay @sbatten 

> 1. Alt does not focus the custom menubar with release notes focused. We do not get the alt event

can we fix that? because it's really strange in my opinion.

**Comment by DanTup** [comment]
@mjbvz is there something webviews need to do to have keys passed back to VS Code? This came up at https://github.com/flutter/devtools/issues/2775 and I can repro in both stable and insiders. The webview hosts an iframe if that matters. I'm unable to interact with VS Code (eg. to open command palette) when focus is in the web view.

**Comment by mjbvz** [comment]
If you can avoid it, don't use an iframe.

If you absolutely must use an iframe, then you need to dispatch the keypresses back to webviews. See https://github.com/microsoft/vscode/issues/65452#issuecomment-586036474
